.github/ISSUE_TEMPLATE/build-error.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Build Error
+about: Report a building error
```

Comment:
Change "Build Error" to "Compile Error"

Reply:
I used "Build Error" to match the one in the item issues.

Reply:
And *I* used "Build Error" in the item template to match the in-game message, to make it more obvious what this is referring to.

Reply:
ok